cdcsdk-testing/src/test/java/com/yugabyte/cdcsdk/testing/TestHelper.java

```diff
@@ -89,68 +104,23 @@ public static String getNewDbStreamId(String dbName) throws Exception {
-            }
-        }));
-        container.withCommand("bin/yugabyted start --listen=0.0.0.0 --master_flags=rpc_bind_addresses=0.0.0.0 --daemon=false");
-        return container;
+    public static GenericContainer<?> getCdcsdkContainerForS3Sink() throws Exception {
+        return new CdcsdkContainer()
+                .withDatabaseHostname(HOST)
+                .withMasterPort(String.valueOf(MASTER_PORT))
+                .withAwsAccessKeyId(System.getenv("AWS_ACCESS_KEY_ID"))
+                .withAwsSecretAccessKey(System.getenv("AWS_SECRET_ACCESS_KEY"))
+                .withAwsSessionToken(System.getenv("AWS_SESSION_TOKEN"))
+                .buildForS3Sink();
     }
 
-    private static Map<String, String> getConfigMap() throws Exception {
-        Map<String, String> configs = new HashMap<>();
-        configs.put("CDCSDK_SOURCE_CONNECTOR_CLASS", "io.debezium.connector.yugabytedb.YugabyteDBConnector");
-        configs.put("CDCSDK_SOURCE_DATABASE_HOSTNAME", HOST);
-        configs.put("CDCSDK_SOURCE_DATABASE_PORT", "5433");
-        configs.put("CDCSDK_SOURCE_DATABASE_MASTER_ADDRESSES", HOST + ":" + MASTER_PORT);
-        configs.put("CDCSDK_SOURCE_DATABASE_SERVER_NAME", "dbserver1");
-        configs.put("CDCSDK_SOURCE_DATABASE_DBNAME", "yugabyte");
-        configs.put("CDCSDK_SOURCE_DATABASE_USER", "yugabyte");
-        configs.put("CDCSDK_SOURCE_DATABASE_PASSWORD", "yugabyte");
-        configs.put("CDCSDK_SOURCE_TABLE_INCLUDE_LIST", "public.test_table");
-        configs.put("CDCSDK_SOURCE_SNAPSHOT_MODE", "never");
-        configs.put("CDCSDK_SOURCE_DATABASE_STREAMID", getNewDbStreamId("yugabyte"));
-
-        // Add configs for the sink
-        configs.put("CDCSDK_SINK_TYPE", "s3");
-        configs.put("CDCSDK_SINK_S3_BUCKET_NAME", "cdcsdk-test");
-        configs.put("CDCSDK_SINK_S3_REGION", "us-west-2");
-        configs.put("CDCSDK_SINK_S3_BASEDIR", "S3ConsumerIT/");
-        configs.put("CDCSDK_SINK_S3_PATTERN", "stream_12345");
-        configs.put("CDCSDK_SINK_S3_FLUSH_RECORDS", "5");
-        configs.put("CDCSDK_SINK_S3_FLUSH_SIZEMB", "200");
-        configs.put("CDCSDK_SERVER_TRANSFORMS", "FLATTEN");
-        configs.put("CDCSDK_SINK_S3_AWS_ACCESS_KEY_ID", System.getenv("AWS_ACCESS_KEY_ID"));
-        configs.put("CDCSDK_SINK_S3_AWS_SECRET_ACCESS_KEY", System.getenv("AWS_SECRET_ACCESS_KEY"));
-        configs.put("CDCSDK_SINK_S3_AWS_SESSION_TOKEN", System.getenv("AWS_SESSION_TOKEN"));
-
-        return configs;
-    }
-
-    public static GenericContainer<?> getCdcsdkContainer() throws Exception {
-        GenericContainer<?> cdcsdkContainer = new GenericContainer<>(DockerImageName.parse("quay.io/yugabyte/cdcsdk-server:latest"));
-
-        // By the time this container is created, the table should be there in the database already
-        cdcsdkContainer.withEnv(getConfigMap());
-        cdcsdkContainer.withNetwork(containerNetwork);
-        cdcsdkContainer.withExposedPorts(8080);
```

Comment:
These three configuration parameters are required for proper startup of CDCSDK server. Otherwise the tests run before the server is ready.